scripts/check-deny.sh

```diff
@@ -11,4 +11,4 @@ cd "$(dirname "${BASH_SOURCE[0]}")/.."
 set -x
 
 cargo install --locked cargo-deny
-cargo deny --all-features --log-level warn check advisories
+cargo deny --all-features --log-level warn check advisories && cargo deny --all-features --log-level warn check licenses
```

Comment:
The default log level seems to be `warn`. I think this could just be:

```
cargo deny check advisories licenses
```

Or really just

```
cargo deny all
```

Really.

Reply:
I'm seeing the check for "bans" failing on master right now:

```
$ cargo deny check all
...
error[B001]: crate 'tokio-util = 0.7.1' is explicitly banned
   ┌─ /Users/kyle.criddle/workspace/vector/deny.toml:55:14
   │
55 │     { name = "tokio-util", version = ">=0.6.9" },
   │              ^^^^^^^^^^^^ banned here


advisories ok, bans FAILED, licenses ok, sources ok
...

Reply:
Ah, hmm, yeah, I see they are failing due to transient dependencies. That doesn't make those bans very useful then 🤔

Reply:
I think we can probably drop the bans since they result in false positives due to transient dependencies. Really we just want to make sure _we_ don't directly depend on the indicated versions of the crates, but I'm not seeing a way to configure `cargo deny` to do that (https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html).

Reply:
If we don't drop the bans I think that would leave us with:

```
cargo deny check advisories licenses sources
```

Reply:
Aaah, I was actually thinking about dropping the bans from the `deny.toml`. They don't seem very useful since running `cargo check bans` flags the transient dependencies (unless you can spot a way to have it only check direct dependencies). Then we could just do `cargo deny all`.

Reply:
Oh!
I see there is this "wrappers" option but it seems to be the opposite of what we want.
https://embarkstudios.github.io/cargo-deny/checks/bans/cfg.html#the-wrappers-field-optional